src/components/Chat/RoomMenu.scss

```diff
@@ -41,6 +41,15 @@
     &:hover {
       background: var(--color-background-chat-toggle-hover);
     }
+    &--on {
+      margin: 0 0.4rem 0.4rem;
+      padding: 0rem;
+      width: 30px;
+    }
+    &--on:hover {
+      background: var(--color-background-chat-toggle-hover);
+      cursor: pointer;
+    }
```

Comment:
this isn't needed... this just duplicates the styles that were already applied. once you get rid of `--on` this will be styled correctly